readme.md

```diff
@@ -136,6 +136,14 @@ The attribute should be an image URL:
 
 Read more about the `media` parameter in the [in the Pinterest documentation](https://developers.pinterest.com/docs/widgets/pin-it/#source-settings).
 
+### Telegram
+
+You can set `data-text` attribute to set a text of the message.
+
+```html
+<div class="telegram" data-text="Hello, world!">Send</div>
```

Comment:
Agree with the `ilyabirman.net` point.

Does Ilya use the “design changes the world” slogan anywhere? I don’t really like it because it sounds like a _приевшийся штамп_ to me, but if Ilya does use it, it’s 👍.